README.md

```diff
@@ -249,7 +258,15 @@ spec:
 ```
 
 Avaiable value for logmode is "devel", "development", "prod", "production".
-The first two work the same set to DEBUG level; the later two work the same, using ERROR level.
+The first two work the same set to DEBUG level; the later two work the same as using ERROR level.
+
+| .spec.devFlags.logmode | stacktrace level | verbosity | Output   | Comments       |
+| ---------------------- | ---------------- | --------- | -------- | -------------- |
+| devel                  | WARN             | INFO      | Console  | lowest level, using epoch time  |
```

Comment:
Frankly, I'm confused here about verbosity level. So, by default it set to INFO ConfigLoggers. Then it can be changed by command line to operator or devflag in DSCI for components to debug, or how?

Reply:
i would say normally, user should not change anything,
and it uses info for both controllers and components.

but when something happens or user prefer more/less logging:
- if they want the change to be on controller: set the flag to operator, either in CSV or community user might wanna build their own image so they change code/config
- if they want the change to be on component: set the devflag.logmode in DSCI.  => i see this is more useful than the first one ^ esp. if it is running in production, a neat json for ETL, or a plaintext for debugging with stacktrace.

Reply:
> i would say normally, user should not change anything, and it uses info for both controllers and components.
> 
> but when something happens or user prefer more/less logging:
> 
> * if they want the change to be on controller: set the flag to operator, either in CSV or community user might wanna build their own image so they change code/config
> * if they want the change to be on component: set the devflag.logmode in DSCI.  => i see this is more useful than the first one ^ esp. if it is running in production, a neat json for ETL, or a plaintext for debugging with stacktrace.

But in the doc will the "verbosity" for "devel" here other than "INFO"?

Reply:
not for what we already have in the code.
if INFO is not enough, (i tested DEBIG, it was loads out to stdout.) we can extend it in the future. 
but for now i will just keep INFO as the only verbositiy for devel, "" and "prod"